pkg/resources/kafka/configmap.go

```diff
@@ -180,12 +230,33 @@ func appendListenerConfigs(advertisedListenerConfig []string, id int32,
 	return advertisedListenerConfig
 }
 
-func generateStorageConfig(sConfig []v1beta1.StorageConfig) string {
+func getMountPathsFromBrokerConfigMap(configMap *v1.ConfigMap) []string {
+	if configMap == nil {
+		return nil
+	}
+	brokerConfig := configMap.Data[kafkautils.ConfigPropertyName]
+	brokerConfigsLines := strings.Split(brokerConfig, "\n")
```

Comment:
Why don't you use `properties.NewProperties()` utility to parse `brokerConfig` and extract the log dir configuration using its `Get` method?